plugins/core/hu.bme.mit.gamma.rcp/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This RCP plugin is pretty cool, but it is definitely not `core`. We should move it into a separate subfolder next to `core` named `rcp`.